content/communities/uswds.md

```diff
@@ -0,0 +1,41 @@
+---
+url: /communities/uswds/
+uid: uswds
+title: 'U.S. Web Design System Community of Practice'
+summary: 'The federal government’s community for news and updates about the U.S. Web Design System'
+featured_image:
+  uid: uswds-2-illio-built-to-grow-alt
+  alt: ''
+
+---
+
+{{< img-right src="uswds-logo" >}}
+
+The U.S. Web Design System (USWDS) is a library of code, tools, and guidance to help government teams design and build fast, accessible, mobile-friendly government websites backed by user research and modern best practices. [Read an overview »](https://designsystem.digital.gov/whats-new/updates/2019/04/08/introducing-uswds-2-0/)
+
+We’re proud that USWDS currently powers nearly [200 federal websites](https://designsystem.digital.gov/getting-started/showcase/all/) and we’re committed to making the design system work better for any federal website. With USWDS 2.0, we’re introducing a design system that’s built to grow with your needs — to help teams build better, more engaging websites for the American public.
+
+## Join
+Send an email to [uswds-subscribe-request@listserv.gsa.gov](mailto:uswds-subscribe-request@listserv.gsa.gov) with an empty subject line and empty email body.
```

Comment:
replace with: `Sign up to join the #uswds-public TTS Slack channel at  [chat.18f.gov](https://chat.18f.gov/)`

Reply:
For now, we're considering this the main community home

Reply:
Done. I added in some additional copy about joining Slack, and what to expect.